.whitesource

```diff
@@ -0,0 +1,19 @@
+{
+  "scanSettings": {
+    "configMode": "AUTO",
+    "configExternalURL": "",
+    "projectToken": "",
+    "baseBranches": []
+  },
+  "checkRunSettings": {
+    "vulnerableCheckRunConclusionLevel": "failure",
```

Comment:
@sfc-gh-jfan , setting this level to "failure" means the branch cannot be merged to master unless by administrative override according to this [documentation:](https://whitesource.atlassian.net/wiki/spaces/WD/pages/697696422/WhiteSource+for+GitHub.com#Completed-with-Failure-Conclusion). But what if there are whitesource vulnerabilities that are ongoing and have no straightforward solution (such as when a library version update requires a java version update, which would break existing customers using older java versions)? I'm concerned this check would then fail for every new PR and then no one could merge any code in JDBC except by administrative override.

We have one of those issues now:https://saas.whitesourcesoftware.com/Wss/WSS.html#!libraryDetails;uuid=79edd1e6-792e-4a6c-b7e4-2b000e4847ad;project=2474172;orgToken=df62f498-e8d7-45d3-a1a0-447176e18c37

Reply:
Hey, @mknister  set this level to `failure` will NOT block the merge unless a `branch protection` rule is setup for whitesource PR check according to whitesource doc. I believe the `branch protection` rule has not yet setup (it can only be configured by repo admin). So the PR check is not going to be a `required` merge gate. 
And the PR check only works on new security vulnerabilities, it won't be blocked by existing security vulnerability. 
Hope that clarifies